include/usb/usb_common.h

```diff
@@ -100,10 +95,13 @@
  * D5:Remote Wakeup -> 0,
  * D4...0:Reserved -> 0
  */
-#define USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP  0x20
-#define USB_CONFIGURATION_ATTRIBUTES                0xC0       \
-	| (COND_CODE_1(CONFIG_USB_DEVICE_REMOTE_WAKEUP,        \
-	   (USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP), (0)))
+#define USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP	BIT(5)
```

Comment:
I will consider one more thing: 
```diff
diff --git a/include/usb/usb_common.h b/include/usb/usb_common.h
index c3c0058e2e..7aa98c3d8e 100644
--- a/include/usb/usb_common.h
+++ b/include/usb/usb_common.h
@@ -95,13 +95,9 @@
  * D5:Remote Wakeup -> 0,
  * D4...0:Reserved -> 0
  */
-#define USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP	BIT(5)
-#define USB_CONFIGURATION_ATTRIBUTES_SELF_POWERED	BIT(6)
 #define USB_CONFIGURATION_ATTRIBUTES BIT(7) \
-	| ((COND_CODE_1(CONFIG_USB_SELF_POWERED, \
-	   (USB_CONFIGURATION_ATTRIBUTES_SELF_POWERED), (0)))   \
-	| (COND_CODE_1(CONFIG_USB_DEVICE_REMOTE_WAKEUP,         \
-	   (USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP), (0))))
+	| (COND_CODE_1(CONFIG_USB_SELF_POWERED, (BIT(6)), (0)))   \
+	| (COND_CODE_1(CONFIG_USB_DEVICE_REMOTE_WAKEUP, (BIT(5)), (0)))
 
 /* Classes */
 #define COMMUNICATION_DEVICE_CLASS	0x02

```
As those macros are not used anywhere else.

Reply:
No please do not remove this defines, it makes code more readable also could be used in the stack or application (it is public API ).